build_tools/pkgci/external_test_suite/sdxl_scheduled_unet_cpu_llvm_task.json

```diff
@@ -0,0 +1,23 @@
+{
+    "config_name": "cpu_llvm_task",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=llvm-cpu",
+      "--iree-llvmcpu-target-cpu-features=host",
+      "--iree-llvmcpu-distribution-size=32"
+    ],
+    "iree_run_module_flags": [
+      "--device=local-task",
+      "--parameters=model=real_weights.irpa",
+      "--module=sdxl_scheduled_unet_pipeline_fp16_cpu.vmfb",
+      "--input=1x4x128x128xf16=@inference_input.0.bin",
+      "--input=2x64x2048xf16=@inference_input.1.bin",
+      "--input=2x1280xf16=@inference_input.2.bin",
+      "--input=1xf16=@inference_input.3.bin",
+      "--expected_output=1x4x128x128xf16=@inference_output.0.bin",
+      "--expected_f16_threshold=0.8f"
```

Comment:
These flags usually live with the model itself and not as part of the config file, but the "scheduled" part of this model / test case is a bit special. Let's keep an eye on that and aim to refactor later. Could file an issue in the test suite repo.